specifyweb/specify/views.py

```diff
@@ -105,19 +124,23 @@ def delete_blockers(request, model, id):
             }
         ] for field, sub_objs in collector.delete_blockers
     ])
-    return http.HttpResponse(api.toJson(result), content_type='application/json')
+    return http.HttpResponse(api.toJson(result),
+                             content_type='application/json')
+
 
 def flatten(l):
     return [item for sublist in l for item in sublist]
 
+
 @login_maybe_required
 @require_http_methods(['GET', 'HEAD'])
 def rows(request, model):
     "Returns tuples from the table for <model>."
     return api.rows(request, model)
 
+
 @require_http_methods(['GET', 'HEAD'])
-@cache_control(max_age=365*24*60*60, public=True)
+@cache_control(max_age=365 * 24 * 60 * 60, public=True)
```

Comment:
(optional) define global constants like `DAY`, `YEAR` ... and use them all around the back-end?